CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ the detailed section referring to by linking pull requests or issues.
 
 - Implemented markdown for asset descriptions
 - Implemented list view for Broker UI
+- Asset Metadata is now editable
```

Comment:
Should we cover this grey vertical bar with the background? In my opinion it looks weird if it crosses into the rectangle.

![image](https://github.com/sovity/edc-ui/assets/75735874/20ccbc52-4f6e-46e2-ab4f-de4749c58b72)

Reply:
asset edit dialog -> asset creation wizard rework is coming soon